include/TGRSIRunInfo.h

```diff
@@ -247,6 +250,8 @@ class TGRSIRunInfo : public TObject {
       double   fAddBackWindow;        // Time used to build Addback-Ge-Events for TIGRESS/GRIFFIN.   (default =150 ns (15.0))
       bool     fIsMovingWindow;       // if set to true the event building window moves. Static otherwise.
       
+	  bool 	   fWaveformFitting;	  // If true, waveform fitting with SFU algorithm will be performed
+
```

Comment:
Can you update the ClassDef to be `ClassDef(TGRSIRunInfo,7)` so that people don't get warnings about the missing waveform fitting variable on their old data when many of us don't need to care about it? You may also have to change the TGRSIRunInfo::Streamer function in order to get it to read and write properly. You should also do a check for version 7 there as well.